src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -441,9 +441,10 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
     case Event(message: CallStartMessage, data) if !data.isPending(message.callKey) =>
       val updatedData = message.handleStatusPersist(this, data)
       val callKey = message.callKey
-
       fetchLocallyQualifiedInputs(callKey) match {
-        case Success(callInputs) => startActor(callKey, callInputs, message.startMode)
+        case Success(callInputs) =>
+          globalDataAccess.updateCallInputs(workflow.id, callKey, callInputs)
```

Comment:
This fires off the DB update async of starting the actor, it feels like those should be composed (and the possibility of error handled).